stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/channel/add/AddChannelFragment.kt

```diff
@@ -38,6 +39,11 @@ class AddChannelFragment : Fragment() {
         bindAddChannelView()
     }
 
+    override fun onResume() {
+        super.onResume()
+        useAdjustResize()
```

Comment:
Can we remove android:windowSoftInputMode="adjustResize" from the manifest now?

Reply:
The really sad part is that this kind of dynamic resizing doesn't work if you don't have that in the manifest. I tried for a while, but it seems to be required, super weird. ([Same conclusion in this SO answer](https://stackoverflow.com/a/47629013/4465208))

Reply:
Facepalm